### ISSUE TITLE
[Racial] Shadowmeld isn't clearing combat / resetting enemies (open world)

### ISSUE THREAD

**Issue by Doppiu** [SallyWhitemane/Mograine#237]
**Description:**
Shadowmeld isn't working as intended in regard to dropping enemies attacking the player and removing them from combat. I will add Shadowmeld does work at 80 (dropping combat at least, as well as shadowmelding spells like death coil and other spells with traveltime).

**How to reproduce:**
Pull some enemies, shadowmeld, they will continue to attack. (This was tested with a level 5 night elf druid on enemies who were level 4/5/6/7)

**How it should work:**
Shadowmeld should drop combat and cause tapped enemies on the player to become untapped, running back to their original position or fixating on another player who previously attacked them.

https://media1.giphy.com/media/9wmsW27mv6qNygd8g9/giphy.gif

**Edit:** After more testing I can see that Shadowmeld actually '_works_', and resets mobs but only if I tag and instantly shadowmeld, the issue seems to be tied to the range of the enemy and the grace system (or lack of) in regards to range. I made a similar ticket a couple of months back about enemy agro range and it's weird calculations of agro range based on levels (https://github.com/SallyWhitemane/Mograine/issues/21).

So bottom line, Shadowmeld works and resets enemies and drops combat, but only if the player is at almost max range (depending on the level gap) from the enemy. For example if I'm level 7, I pull an enemy that's level 7 and shadowmeld almost instantly after pulling, it works as intended. If I let that enemy get within <15-20 yards of me and Shadowmeld, it doesn't work. The range calculation based on level seems very off since meld **_should_** almost always work for enemies within 0 - (n) yards unless the player is substantially lower than the enemy (10+ levels).
